Giovaniavs/MyProject-Client
# Feature/home

## ***What I Did:***
- Created the Home section with the effects and components.

## ***How to Test:***
1. Run **yarn start** and see the first component in your screen.
2. After the **step 1**, please, test in different resolutions to see the responsivity working!
